add OneDegree (#2)
yiochou/Resume: README.md

<!-- The unordered list immediately after the h1 will be formatted on a single
line. It is intended to be used for contact details -->

- (+886)927309966

> - hi@yiochou.com

Reviewer: 水喔
你用什麼 mail server？

Author: [improvmx](https://improvmx.com/) 這間

<!-- The paragraph after the h1 and ul and before the first h2 is optional. It
is intended to be used for a short summary. -->

---

## Work Experience

<!-- You have to wrap the "left" and "right" half of these headings in spans by
hand -->

### <span> OneDegree, Backend Engineer </span> 2021/10 - present

> ##### Tech: Python, PostgresSQL, SQLAlchemy

Reviewer: ```suggestion
##### Tech: Python, PostgreSQL, SQLAlchemy
```

> - Led the team of 4 backend engineers. Planned and Assigned tasks to members. Coordinated with them to have good teamework and improve efficiency.

Reviewer: ```suggestion
- Led a team of 4 backend engineers. Planned and assigned tasks to members. Coordinated with them to achieve effective teamwork and improve efficiency.
```
good 有點弱，改成 effective，和 improve efficiency 稍微重疊，看要不要拿掉，換成更具體的事證？

Author: 說提升 scrum 的 velocity 不知道有沒有道理 XD

> - Rejuvenated core insurance modules progressively for better product usability. Developed flexible rule engine that user can write python code to calculate and make decisions.

Reviewer: to calculate and make decisions 這裡意思不太清楚，這樣改有符合原意嗎？
```suggestion
- Rejuvenated core insurance modules progressively for better product usability. Developed flexible rule engine for users to easily utilize the modules in python script.
```

Author: 有！ 更 general 的說法！

> - Clarified numerous legacy codes with PM and senior members. Scheduled tasks and cleaned up these codes step by step.

Reviewer: 提到 schedule tasks 所以是你在帶領這個大掃除嗎？
```suggestion
- Drove the annual/quarterly codebase clean-up. Clarified numerous legacy code with PM and senior members.
```

Author: 算是啦 XD 
不算 annual/quarterly 耶，是每個 sprint 排一些進去清
